internal/renderers/skia/Cargo.toml

```diff
@@ -46,7 +46,7 @@ glow = { version = "0.12" }
 unicode-segmentation = { version = "1.8.0" }
 
 glutin = { version = "0.30", default-features = false, features = ["egl", "wgl"] }
-winit = { version = "0.28.2", optional = true, default-features = false }
+winit = { version = "=0.28.3", optional = true, default-features = false }
```

Comment:
Why is this pinned?

Reply:
Ah, I think I was too defensive here. You're right, that's not needed - I'll remove it.